composer.json

```diff
@@ -117,6 +117,9 @@
             "export-configuration": "drush config-export --yes"
         },
         "drupal-scaffold": {
+            "locations": {
+                "web-root": "web/"
+            },
             "source": "https://raw.githubusercontent.com/pantheon-systems/drops-8-scaffolding/{version}/{path}",
```

Comment:
instead of this... consider using https://github.com/pantheon-systems/drupal-integrations. Thsi will simplify what all you need to do for scaffolding. Your composer.json would look more like this: https://github.com/pantheon-systems/drupal-composer-managed/blob/default/composer.json#L33